src/test/java/seedu/address/logic/parser/HustleBookParserTest.java

```diff
@@ -36,20 +36,20 @@
     @Test
     public void parseCommand_add() throws Exception {
         Person person = new PersonBuilder().build();
-        AddCommand command = (AddCommand) parser.parseCommand(PersonUtil.getAddCommand(person));
+        AddCommand command = (AddCommand) parser.parseCommand(PersonUtil.getAddCommand(person), null);
```

Comment:
It would be better if `null` is not used as this can cause unexpected null errors in the future. Maybe you can modify the way the method works?